src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -293,8 +293,11 @@ cdef class Blob:
     @classmethod
     def from_file_path(cls, filename: Union[str, Path]):
         cdef bytes packed = os.fsencode(filename)
+        cdef hb_blob_t* blob = hb_blob_create_from_file_or_fail(<char*>packed)
+        if blob == NULL:
+            raise FileNotFoundError()
```

Comment:
I’m not sure if this is the right exception since it can also fail due to memory allocation or other issues, but I’m not sure what generic exception is better here or if I should make a new one.

Reply:
Maybe a new `UHarfBuzzException`? I think it's appropriate to have library errors.